example/ios/Podfile

```diff
@@ -24,6 +24,7 @@ end
 target 'WaasSdkReactNativeExample' do
   config = use_native_modules!
 
+  pod 'WaasSdk', :path => "../../WaasSdk.podspec"
```

Comment:
the example app, even if it's using react-native, declares an explicit dependency on `WaasSdk`.

This is to let the return types from WaasSdk be accessible within the RN app. If you don't add this you get very weird bugs.